pom.xml

```diff
@@ -9,6 +9,39 @@
     <version>1.16.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.0.0-M3</version>
+                <executions>
+                    <execution>
+                        <id>enforce-maven</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireMavenVersion>
+                                    <version>3.6.0</version>
+                                </requireMavenVersion>
+                            </rules>    
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>    
+  
+    <repositories>
+        <repository>
+            <id>central</id>
+            <name>Central Repository</name>
+            <url>https://repo.maven.apache.org/maven2/ </url>
+        </repository>
+    </repositories>
+
```

Comment:
I do not mind limiting the version of the maven at the program level (by the way, what kind of error will we get in the logs?)

I do not like the idea of adding `https://repo.maven.apache.org/maven2/` in pom. That was fixed in Maven 3.2.3,  5 years ago, so I guess that such old version is used very rarely

Reply:
Perhaps removing the explicit reference to Maven Central with SSL is the way to go - you are right, most people are likely not running such an old version of Maven which makes this PR and changes unnecessary.

FYI: the output of using an older build of Maven with these enforcer settings is described here:
https://github.com/OHDSI/ArachneCommons/issues/225#issuecomment-580375123.

I'd propose that we close this PR and the other issue since no changes are required here. If we find that the OHDSI community has challenges due to older versions of Maven, then we can revisit this.